cocos/platform/CCPlatformConfig.h

```diff
@@ -154,6 +154,7 @@ THE SOFTWARE.
 #if (CC_TARGET_PLATFORM == CC_PLATFORM_WIN32)
 #ifndef __MINGW32__
 #pragma warning (disable:4127) 
+#pragma warning (disable:4018)
```

Comment:
too many files contanins this warning..difficult on close all that.a easy  is disable this warning